multigen/pipes.py

```diff
@@ -866,8 +870,8 @@ def setup(self, fimage, width=None, height=None,
         if self.model_type == ModelType.FLUX and hasattr(cscales, '__len__'):
             cscales = cscales[0] # multiple controlnets are not yet supported
         self.pipe_params.update({
-            "width": image.size[0] if width is None else width,
-            "height": image.size[1] if height is None else height,
+            "width": image.width if width is None else width,
```

Comment:
Can `image` have either `size` or `width`/`height` fields depending on its source? E.g., if the image is loaded from the source file using PIL, it'll have `width`/`height`. If it comes from automatic processing with numpy, it'll have `size`?